azure-keyvault/azure/keyvault/generated/__init__.py

```diff
@@ -1,18 +1,149 @@
+from ..models import AdministratorDetails as __models_AdministratorDetails
+from ..models import OrganizationDetails as __models_OrganizationDetails
+from ..models import IssuerAttributes as __models_IssuerAttributes
+from ..models import IssuerBundle as __models_IssuerBundle
+from ..models import Contact as __models_Contact
+from ..models import Contacts as __models_Contacts
+from ..models import KeyCreateParameters as __models_KeyCreateParameters
+from ..models import KeyImportParameters as __models_KeyImportParameters
+from ..models import KeyOperationsParameters as __models_KeyOperationsParameters
+from ..models import KeySignParameters as __models_KeySignParameters
+from ..models import KeyVerifyParameters as __models_KeyVerifyParameters
+from ..models import KeyUpdateParameters as __models_KeyUpdateParameters
+from ..models import KeyRestoreParameters as __models_KeyRestoreParameters
+from ..models import SecretSetParameters as __models_SecretSetParameters
+from ..models import SecretUpdateParameters as __models_SecretUpdateParameters
+from ..models import CertificateCreateParameters as __models_CertificateCreateParameters
+from ..models import CertificateImportParameters as __models_CertificateImportParameters
+from ..models import CertificateUpdateParameters as __models_CertificateUpdateParameters
+from ..models import CertificateMergeParameters as __models_CertificateMergeParameters
+from ..models import CertificateIssuerSetParameters as __models_CertificateIssuerSetParameters
+from ..models import CertificateIssuerUpdateParameters as __models_CertificateIssuerUpdateParameters
+from ..models import CertificateOperationUpdateParameter as __models_CertificateOperationUpdateParameter
+from ..models import KeyOperationResult as __models_KeyOperationResult
+from ..models import KeyVerifyResult as __models_KeyVerifyResult
+from ..models import BackupKeyResult as __models_BackupKeyResult
+from ..models import PendingCertificateSigningRequestResult as __models_PendingCertificateSigningRequestResult
+from ..models import KeyVaultError as __models_KeyVaultError
+from ..models import KeyVaultErrorException as __models_KeyVaultErrorException
+from ..models import KeyItemPaged as __models_KeyItemPaged
+from ..models import SecretItemPaged as __models_SecretItemPaged
+from ..models import CertificateItemPaged as __models_CertificateItemPaged
+from ..models import CertificateIssuerItemPaged as __models_CertificateIssuerItemPaged
+from ..models import JsonWebKeyType as __models_JsonWebKeyType
+from ..models import KeyUsageType as __models_KeyUsageType
+from ..models import ActionType as __models_ActionType
+from ..models import JsonWebKeyOperation as __models_JsonWebKeyOperation
+from ..models import JsonWebKeyEncryptionAlgorithm as __models_JsonWebKeyEncryptionAlgorithm
+from ..models import JsonWebKeySignatureAlgorithm as __models_JsonWebKeySignatureAlgorithm
+
+import warnings
+
+warnings.warn("deprecated", DeprecationWarning)
```

Comment:
Message should be have more information like "import azure.keyvault.generated is now deprecated and will disappear in future version"